src/mkpsxiso/main.cpp

```diff
@@ -512,19 +617,50 @@ int Main(int argc, char* argv[])
 
 					if ( !global::NoIsoGen )
 					{
-						int trackLBA = writer.SeekToEnd();
+						int trackLBA = writer.SeekToEnd(); //Fix me to LBA from data tracks?

```

Comment:
This will have to go once I have memory-mapped file writing done for the ISO writer (for concurrent async ECC/EDC calculations), so don't worry about it yet.

Reply:
New comment removed in G4Vi@48bec80